CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 2.0.0 / 2023-02-02
+### Features
+- Feat: Support adding annotations to the fluent-operator deployment (#467)
+- Feat: Support adding labels to the fluent-operator and the fluent-bit pods (#468)
+- Feat: Add external plugin flag in the Fluent-Bit watcher (#469)
+- Feat: Support adding annotations to the fluent-bit DaemonSet (#474)
+- Feat: Add the `Collector` CRD and controller to support deploying Fluent Bit as a StatefulSet (#484)
+- Feat: Add process termination timeout to fluent-bit-watcher (#512) 
+- Feat: Add `dnsPolicy` and other Kubernetes filter options to the FluentBit CRD (#528) 
+
+### ENHANCEMENT
+- Add the `DockerModeParser` parameter to the fluentbit tail plugin (#486)
+- Dependabot: Update schedule and fix typo (#493)
```

Comment:
`Dependabot: Update schedule and fix typo` dependabot's PR should be moved to the end